api/app/langchain/tools/util/handleTools.test.js

```diff
@@ -11,21 +11,20 @@ var mockPluginService = {
 };
 
 
-jest.mock('../../../models/User', () => {
+jest.mock('../../../../models/User', () => {
   return function() {
     return mockUser;
   };
 });
 
-jest.mock('../../../server/services/PluginService', () => mockPluginService);
+jest.mock('../../../../server/services/PluginService', () => mockPluginService);
```

Comment:
Just a thought, but it would be nice if we could have aliases like we do for the api to avoid getting stuck in dot-dot-slash hell. Obviously there is no bundler but there has to be a better way to navigate the folder structure.

Reply:
I take that back. There's other ways of doing it, but none that are as convenient as just setting aliases unforuntately.